src/main/java/com/autotune/common/data/result/AggregationInfoResult.java

```diff
@@ -15,13 +15,32 @@
  *******************************************************************************/
 package com.autotune.common.data.result;
 
+import com.autotune.utils.AutotuneConstants;
+import org.json.JSONObject;
+
 public class AggregationInfoResult {
     private Double sum;
     private Double avg;
     private Double min;
     private Double max;
     private String format;
```

Comment:
Also add `mode`, `median`, `count` and `range` and have them listed alphabetically

Reply:
Added